spec/numberformat.html

```diff
@@ -458,18 +535,125 @@ <h1>Properties of Intl.NumberFormat Instances</h1>
+      <li>[[TrailingZeroDisplay]] is one of the String values *"auto"* or *"stripIfInteger"*, indicating whether to strip trailing zeros if the formatted number is an integer (i.e., has no non-zero fraction digit).</li>
     </ul>
 
+    <emu-table id="table-intl-rounding-modes">
+      <emu-caption>Rounding modes in Intl.NumberFormat</emu-caption>
+      <table class="real-table">
+        <thead>
+          <tr>
+            <th rowspan="2">Identifier</th>
+            <th rowspan="2">Description</th>
+            <th colspan="5">Examples: Round to 0 fraction digits</th>
+          </tr>
+          <tr>
+            <th>-1.5</th>
+            <th>0.4</th>
+            <th>0.5</th>
+            <th>0.6</th>
+            <th>1.5</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>*"ceil"*</td>
+            <td>Toward positive infinity</td>
+            <td>➕ [-1]</td>
+            <td>➕ [1]</td>
+            <td>➕ [1]</td>
+            <td>➕ [1]</td>
+            <td>➕ [2]</td>
+          </tr>
+          <tr>
+            <td>*"floor"*</td>
+            <td>Toward negative infinity</td>
+            <td>➖ [-2]</td>
+            <td>➖ [0]</td>
+            <td>➖ [0]</td>
+            <td>➖ [0]</td>
+            <td>➖ [1]</td>
+          </tr>
+          <tr>
+            <td>*"expand"*</td>
+            <td>Away from zero</td>
+            <td>➖ [-2]</td>
+            <td>➕ [1]</td>
+            <td>➕ [1]</td>
+            <td>➕ [1]</td>
+            <td>➕ [2]</td>
+          </tr>
+          <tr>
+            <td>*"trunc"*</td>
+            <td>Toward zero</td>
+            <td>➕ [-1]</td>
+            <td>➖ [0]</td>
+            <td>➖ [0]</td>
+            <td>➖ [0]</td>
+            <td>➖ [1]</td>
+          </tr>
+          <tr>
+            <td>*"halfCeil"*</td>
+            <td>Ties toward positive infinity</td>
+            <td>➕ [-1]</td>
+            <td>➖ [0]</td>
+            <td>➕ [1]</td>
+            <td>➕ [1]</td>
+            <td>➕ [2]</td>
+          </tr>
+          <tr>
+            <td>*"halfFloor"*</td>
+            <td>Ties toward negative infinity</td>
+            <td>➖ [-2]</td>
+            <td>➖ [0]</td>
+            <td>➖ [0]</td>
+            <td>➕ [1]</td>
+            <td>➖ [1]</td>
+          </tr>
+          <tr>
+            <td>*"halfExpand"*</td>
+            <td>Ties away from zero</td>
+            <td>➖ [-2]</td>
+            <td>➖ [0]</td>
+            <td>➕ [1]</td>
+            <td>➕ [1]</td>
+            <td>➕ [2]</td>
+          </tr>
+          <tr>
+            <td>*"halfTrunc"*</td>
+            <td>Ties toward zero</td>
+            <td>➕ [-1]</td>
+            <td>➖ [0]</td>
+            <td>➖ [0]</td>
+            <td>➕ [1]</td>
+            <td>➖ [1]</td>
+          </tr>
+          <tr>
+            <td>*"halfEven"*</td>
+            <td>Ties toward even cardinality</td>
```

Comment:
Even cardinality of what?
```suggestion
            <td>Ties toward an even rounding increment multiple</td>
```